app/gui/qt/lang/sonic-pi_fr.ts

```diff
@@ -652,7 +651,7 @@ Veuillez nous excuser, une erreur critique s&apos;est produite pendant le démar
     <message>
         <location filename="../mainwindow.cpp" line="2038"/>
         <source>Improve readability of code</source>
-        <translation>Amélioration de la lisibilité du code</translation>
+        <translation>Améliore de la lisibilité du code</translation>
```

Comment:
[Nicolas Dermine]
> s/Améliore de/Améliore ?

No idea.  I'm just passing on the translation done by people on Weblate,
I do not know French. :)

If you  know French, please review and see if any of the fixes are
useful. :)
-- 
Happy hacking
Petter Reinholdtsen